ctapipe_io_magic/laser_params_corr_test.py

```diff
@@ -0,0 +1,411 @@
+                    elif variable == "stime232":
+                        stime232 = int(value)
+                    elif variable == "stime233":
+                        stime233 = int(value)
+                    elif variable == "stime234":
+                        stime234 = int(value)
+                    elif variable == "stime27":
+                        stime27 = int(value)
+                    elif variable == "stime285":
+                        stime285 = int(value)
+                    elif variable == "gkC_0Period1":
+                        gkC_0Period1 = float(value)
+                    elif variable == "gkFullOverlap5":
+                        gkFullOverlap5 = float(value)
+                    elif variable == "gkFullOverlap6":
+                        gkFullOverlap6 = float(value)
+                    elif variable == "gkFullOverlap7":
+                        gkFullOverlap7 = float(value)
+                    elif variable == "gkIntegrationWindow":
+                        gkIntegrationWindow = float(value)
+                    elif variable == "gkHWSwitchV1to4":
+                        gkHWSwitchV1to4 = int(value)
+
+        C_0 = gkC_0Period1  # absolute calibration at beginning of 2013
+        if stime21 > 0:
+            fFullOverlap = gkFullOverlap5  # overlap has moved, after installation of new DAQ, seems that saturation occurs earlier now
+        if stime22 > 0:
+            fFullOverlap = 400.  # overlap has moved, after installation of new DAQ, seems that saturation occurs earlier now
+        if stime232 > 0:
+            fFullOverlap = gkFullOverlap6  # overlap has moved, after installation of new DAQ, seems that saturation occurs earlier now
+        if stime233 > 0:
+            fFullOverlap = gkFullOverlap5  # overlap has moved, after installation of new DAQ, seems that saturation occurs earlier now
+        if stime234 > 0:
+            fFullOverlap = gkFullOverlap7  # overlap has moved, after installation of new DAQ, seems that saturation occurs earlier now
+        if stime26 > 0:
+            fFullOverlap = gkFullOverlap5  # overlap was corrected again, valid for only one night
+        if stime261 > 0:
+            fFullOverlap = gkFullOverlap7  # overlap got worse, probably screw loose
+        if stime27 > 0:
+            fFullOverlap = 1500.  # overlap got worse, probably screw loose
+        if stime285 > 0:
+            fFullOverlap = 1000.  # overlap got worse, probably screw loose
+        range_max_clouds = 23000.
+
+        zenith_radians = np.deg2rad(zenith)
+
+        if (range_max - 100.) * np.cos(zenith_radians) < range_max_clouds:
+            print("Reducing maximum range to search for clouds from", range_max_clouds, "to",
+                 (range_max - 100.) * np.cos(zenith_radians), "Zd=", zenith)
+            range_max_clouds = (range_max - 100.) * np.cos(zenith_radians)
+        return C_0
+
+    def apply_time_dependent_corrections(self, datime, c0, coszd, case_index=None):
```

Comment:
case_index is not used in the function

add doc string with explanation of parameters